goagen/gen_app/test_generator.go

```diff
@@ -12,11 +12,12 @@ import (
 
 	"github.com/goadesign/goa/design"
 	"github.com/goadesign/goa/goagen/codegen"
+	"github.com/goadesign/goa/goagen/utils"
 )
 
 func makeTestDir(g *Generator, apiName string) (outDir string, err error) {
 	outDir = filepath.Join(g.OutDir, "test")
-	if err = os.RemoveAll(outDir); err != nil {
```

Comment:
Do not change this as it deletes temporary folders created by tests

Reply:
When I don't change it here, then is my .svn directory deleted in app/test. Because of that I changed it also at this place.

Reply:
Sorry you're right, I got confused and thought this was the reason for the build failures.